sda-commons-server-kafka-example/src/test/java/org/sdase/commons/server/kafka/KafkaExampleProducerIT.java

```diff
@@ -47,6 +47,8 @@ class KafkaExampleProducerIT {
   void testUseProducer() throws JsonProcessingException {
     // given
     KafkaExampleProducerApplication application = DW.getApplication();
+    // creating topic manually since the createTopicIfMissing method was removed
```

Comment:
I would not necessarily reference functionality that was removed, might be a bit confusing.